tests/UI/campaigns/functional/BO/14_advancedParameters/08_logs/01_filterSortAndPagination.ts

```diff
@@ -93,14 +88,14 @@ describe('BO - Advanced Parameters - Logs : Filter, sort and pagination logs tab
 
       await dashboardPage.goToSubMenu(page, dashboardPage.advancedParametersLink, dashboardPage.logsLink);
 
-      const pageTitle = await logsPage.getPageTitle(page);
+      const pageTitle: string = await logsPage.getPageTitle(page);
       await expect(pageTitle).to.contains(logsPage.pageTitle);
     });
 
     it('should check the number of logs', async function () {
       await testContext.addContextItem(this, 'testIdentifier', 'checkLogsNumber', baseContext);
 
-      const numberOfElements = await logsPage.getNumberOfElementInGrid(page);
+      const numberOfElements: number = await logsPage.getNumberOfElementInGrid(page);
```

Comment:
again, prefer adding the type to `logsPage.getNumberOfElementInGrid` return

Reply:
Fixed